src/Pages/Applications/DevtronApps/Details/AppConfigurations/MainContent/DeploymentConfigCompare/DeploymentConfigCompare.tsx

```diff
@@ -251,8 +257,24 @@ export const DeploymentConfigCompare = ({
                 identifierId: chartRefId || compareWithIdentifierId,
                 pipelineId: compareWithPipelineId,
             }),
+        ] as const
+
+        const [secretsData, ...appConfigData] = await Promise.all([
+            !isSuperAdmin ? getCompareSecretsData([...payloads]) : null,
```

Comment:
remove this isSuperAdmin check as discussed

Reply:
As per discussions with @vivek-devtron , if we remove this check, for superAdmins there will redundant api calls. The probability of a person being demoted from super admin to view only user is very slim. So we can expect that the user will at-least refresh the page.